CMake/HighFiveTargetDeps.cmake

```diff
@@ -0,0 +1,61 @@
+# Link against target system libs
+# -------------------------------
+
+# therefore making it possible to have new dependencies each build
+if(NOT TARGET highfive_deps)
+  add_library(highfive_deps INTERFACE)
+else()
+# Reset if imported
+set_target_properties(highfive_deps PROPERTIES
+  INTERFACE_COMPILE_DEFINITIONS ""
+  INTERFACE_COMPILE_OPTIONS ""
```

Comment:
You should still `target_compile_features(HighFive INTERFACE cxx_std_11)` somewhere

Reply:
_target_compile_features_ is in **cmake** >= 3.1 not in 3.0 (the minimum we support).

Reply:
```cmake
if(${CMAKE_VERSION} VERSION_GREATER_EQUAL "3.1") 
    target_compile_features(HighFive INTERFACE cxx_std_11)
endif()
```

Reply:
you can even include and `else()` with old-school behaviour